### ISSUE TITLE
'crisp' option does not disable antialias

### ISSUE THREAD

**Issue by Vanglant** [replit/kaboom#548]
### Version

v2000.2

### What browsers are you seeing the problem on?

Chrome

### What happened?

Hi! Trying out Kaboom today, seems like setting the `crisp` option in kaboom main function does not disable antialias.

### What's the expected behavior?

As the docs state for the `crisp` option: `Disable antialias and enable sharp pixel display.`

Drawing a hollow circle:
<img width="332" alt="image" src="https://user-images.githubusercontent.com/18269928/155207748-048e1fd7-1f85-41e1-b66c-249841aeb10b.png">

Without antialias it should look more pixelated, something like this?
<img width="320" alt="image" src="https://user-images.githubusercontent.com/18269928/155208126-e04edd75-ec35-4333-88d3-bfa2fa0a7880.png">

Thanks!

### Minimum reproducable code

```shell
kaboom({
	crisp: true,
	background: [0,0,0]
})

const hollowCircle = () => {
	return {
		id: 'circleId',
		draw() {
			drawCircle({
		        radius: 100,
		        fill: false,
		        outline: {
		          color: rgb(255, 255, 255),
		          width: 3,
		        },
		    });
		}
	}
}

const player = add([
	hollowCircle(),  
	pos(130, 120)
])
```

**Comment by MeowcaTheoRange** [comment]
Hello,
What `crisp` actually does is disable the **canvas anti-alias**, which makes rendering look a lot better for small/rescaled Kaboom games.
It does not remove **internal anti-alias**, which makes rendering of sprites look nicer and smoother by using sub-pixel math.

TL;DR:
**`crisp` affects the antialiasing of individual pixels, not the antialiasing of pictures represented in those pixels.**
